### PR TITLE
JBPM-5370 - Administration service for processes and tasks - services…

### PR DESCRIPTION
**Pull request by mswiderski**
… api and kie server

depends on https://github.com/droolsjbpm/jbpm/pull/642

### CODE REVIEW

**Comment by sutaakar** [comment]
Looks good for me, one minor idea about tests in comment.

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/admin/UserTaskAdminServiceIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,534 @@
...
+    }
+
+    @Override
+    protected void addExtraCustomClasses(Map<String, Class<?>> extraClasses) throws Exception {
+        extraClasses.put(PERSON_CLASS_NAME, Class.forName(PERSON_CLASS_NAME, true, kieContainer.getClassLoader()));
+    }
+
+    @Test
+    public void testAddRemovePotOwners() throws Exception {
+        changeUser(USER_ADMINISTRATOR);
```
Comment (mswiderski) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/admin/UserTaskAdminServiceIntegrationTest.java]: added @After method to reset the user. Is that what you had in mind?

**Comment by livthomas** [comment]
@mswiderski, I am just curious, why have you decided to put `ProcessAdminServicesClient` and `UserTaskAdminServicesClient` to a new package while all the other clients are directly in `org.kie.server.client` package?

**Comment by mswiderski** [comment]
@livthomas mainly to indicate it's not regular client but an admin one which should be certainly used in caution. Do you see any issue with that?

**Comment by livthomas** [comment]
@mswiderski, ok, it makes sense. No issue, I was just browsing the folders and noticed it is separated but I can see now that `ProcessAdminServicesClient` has already been there.

**Pull request merged**